package.json

```diff
@@ -199,19 +199,17 @@
     "node-addon-loader": "alexlyp/node-addon-loader#master",
     "prop-types": "^15.5.10",
     "qr-image": "^3.2.0",
-    "react": "15",
-    "react-addons-css-transition-group": "^15.6.2",
-    "react-dom": "15",
+    "react": "16",
```

Comment:
Can you specify the patch version as well like `16.0.x` just to have the security/minor fix, please?

Reply:
done

Reply:
Sorry my comment was not accurate enough, can you add `16.0.x` or `~16.0.0`